CHANGELOG.md

```diff
@@ -20,7 +21,7 @@ Desktop App v4.3.0 contains multiple medium level security fixes. [Upgrading](ht
 ### Improvements
 
 #### All Platforms
- - Added support for maintaining a user's online status while the desktop app is in the background but the user is interacting with their computer.
+ - Added support for maintaining a user's online status while the desktop app is in the background but the user is interacting with their computer. Requires Mattermost Server v5.16.0 or later.
```

Comment:
Could add in here that the latest dot releases of 5.14 and 5.15 support the online status feature.

Reply:
Which versions exactly? We can add them here